xml/snmp_sysdescr.xml

```diff
@@ -5158,6 +5158,121 @@ Copyright (c) 1995-2005 by Cisco Systems
    </fingerprint>
 
    <!--======================================================================
+                              Mac OS X
+   =======================================================================-->
+   <fingerprint pattern="^Darwin (\S*) \S* Darwin Kernel Version 7.(\d+).\d+: .*RELEASE_\S* ?\s* (\S* ?\S*)$">
+      <description>Mac OS X 10.3 (Panther)</description>
+      <example host.name="mac-10-3-s-u" os.arch="Power Macintosh" os.version.version="0">Darwin mac-10-3-s-u 7.0.0 Darwin Kernel Version 7.0.0: Wed Sep 24 15:48:39 PDT 2003; root:xnu/xnu-517.obj~1/RELEASE_PPC Power Macintosh</example>
```

Comment:
really minor, but I try to sanitize hostnames when possible, typically using `example.com`